modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -291,6 +291,197 @@ An example of HTTP input for this flow is:
 }
 ----
 
+== Pre-query Example
+
+This example provides metadata for an object to be queried, generates two sub-requests that contain this metadata, and then executes the sub-requests. The following screenshots show the flow for the Pre-Query example:
+
+.The flow for this example includes an HTTP Listener component, Transform components, and multiple Salesfore Composite operations.
+image::salesforce/salesforce-composite-prequery-flow.png[Pre-Query flow in Studio]
+
+=== Start Creating the Flow by Adding HTTP Listener
+
+`HTTP Listener` initiates the flow when it detects an event in the `executeFlow` path:
+
+. Create a new Mule project in Studio.
+. In the Mule Palette view, search for *HTTP* and select the *Listener* operation.
+. Drag the *Listener* operation onto the canvas.
+. In the Listener properties window, set the *Path* field to `/executeFlow`.
+
+=== Add the First Transform Message Component
+
+This `Transform Message` component sets a name for the object to be created and passes the name to the `Pre create` operation:
+
+. In the Mule Palette view, search for *Transform Message*.
+. Drag the *Transform Message* component onto the canvas, to the right of *Listener*.
+. Enter the name `MyNewAccount`:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+%output application/json
+---
+{
+   Name: "MyNewAccount"
+}
+----
+
+=== Add the Pre Create Operation
+
+The `Pre create` operation provides metatdata for creating an object called `NewAccount`. It generates a sub-request for the `executeCompositeBatch` operation:
+
+. In the Mule Palette view, search for *Salesforce* and select the *Salesforce Composite Pre create* operation.
+. Drag the *Pre create* operation to the right of *Transform Message*.
+. Click the green plus icon to the right of the *Connector configuration* field to access the Salesforce Composite global element configuration fields.
+. Complete the fields using variables, as follows:
++
+.Complete fields in the General section to create a reusable global element for Salesforce Composite Connector.
+image::salesforce/salesforce-composite-global-config.png[Salesforce Composite Connector global element configuration]
++
+. In the Pre create properties window, enter `Account` in the *Type* field.
+. In the YAML file `src/main/resources/properties/project_name_dev.yaml`, declare and enter values for the variables in the global element .
```

Comment:
```suggestion
. In the YAML file `src/main/resources/properties/project_name_dev.yaml`, declare and enter values for the variables in the global element.
```
Note use of lowercase "global element," because I think I saw "Global element" in a previous file? See Term Presentation list for guidance.